src/pages/organize/[orgId]/projects/[campId]/areaassignments/[areaAssId]/report.tsx

```diff
@@ -94,216 +111,352 @@ const AreaAssignmentReportPage: PageWithLayout<AreaAssignmentReportProps> = ({
-                      )?.question
-                    }" you need to pick another
-                  yes/no-question to be the question that defines if the msision
+
+                  {metricBeingDeleted?.definesDone ? (
+                    <Typography>
+                      {`If you want to delete "${metricBeingDeleted.question}" you need to pick another
+                  choice question to be the question that defines if the mision
                   was successful`}
-                  </Typography>
-                  <Box display="flex" flexDirection="column" gap={1}>
-                    <Typography>Yes/no questions</Typography>
-                    {assignment.metrics
-                      .filter(
-                        (metric) =>
-                          metric.kind == 'boolean' &&
-                          metric.id != idOfMetricBeingDeleted
-                      )
-                      .map((metric) => (
-                        <Box
-                          key={metric.question}
-                          alignItems="center"
-                          display="flex"
-                          gap={1}
-                          justifyContent="space-between"
-                          width="100%"
-                        >
-                          {metric.question}
-                          <Button
-                            onClick={() => {
-                              if (idOfMetricBeingDeleted) {
-                                const filtered = assignment.metrics.filter(
-                                  (metric) =>
-                                    metric.id != idOfMetricBeingDeleted
-                                );
-                                updateAreaAssignment({
-                                  metrics: [
-                                    ...filtered.slice(
-                                      0,
-                                      filtered.indexOf(metric)
-                                    ),
-                                    {
-                                      ...metric,
-                                      definesDone: true,
-                                    },
-                                    ...filtered.slice(
-                                      filtered.indexOf(metric) + 1
-                                    ),
-                                  ],
-                                });
-                              }
-                              setAnchorEl(null);
-                              setIdOfQuestionBeingDeleted(null);
-                            }}
-                            variant="outlined"
+                    </Typography>
+                  ) : (
+                    <Typography>
+                      Are you sure you want to delete this question? This action
+                      is permanent and it cannot be undone.
+                    </Typography>
+                  )}
+
+                  {metricBeingDeleted?.definesDone ? (
```

Comment:
Personally, I don't like these very long ternary expressions (where `expr-when-true` and `expr-when-false` are very many lines long). I find it very difficult to read what is part of the ternary and what isn't, especially when it's so long that it can't fit all at once on the screen.

I prefer using two conditions, i.e.:

```tsx
{metricBeingDeleted?.definesDone && (
  // content when true
)}
{!metricBeingDeleted?.definesDone && (
  // content when false
)}
```